content/management/v1/core-resources/collaborators/remove-users-with-sso.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Remove user with SSO 
```

Comment:
@iamfortune I would suggest moving this into the `delete-collaborator.md` just adding the `sso_id` as a type of `id` that can be used to delete a collaborator. It's just a different type of id, so I don't think we need a separate piece of docs for this.